src/api/messages/getTemporaryFileUrl.js

```diff
@@ -0,0 +1,29 @@
+/* @flow strict-local */
+import type { Auth, ApiResponseSuccess } from '../transportTypes';
+import { apiGet } from '../apiFetch';
+
+type ApiResponseTempFileUrl = {|
+  ...ApiResponseSuccess,
+  url: string,
+|};
+
+/**
+ * Returns a temporary, partial URL to a realm upload that does not require
```

Comment:
>Hmm, can we squeeze this first bit into a single line? 

Sure. I changed it to the imperative form now. I don't see any mention of line width in our style guide (docs/style.md), so I assumed it was 80 characters, and so I had moved it to a new line.
(But our Prettier config (.prettierrc) says 100 characters.)

Reply:
Done.

Reply:
> Sure. I changed it to the imperative form now. I don't see any mention of line width in our style guide (docs/style.md), so I assumed it was 80 characters, and so I had moved it to a new line.
> (But our Prettier config (.prettierrc) says 100 characters.)

In general we want to keep prose to a width like 70 characters, for the same reason as in commit messages. Code can go wider, because you don't generally read it in the same way.